src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -2472,4 +2472,41 @@
     </Environment>
   </Provider>
 
+  <!--
+                                                ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                                ██░▄▄░█░▄▄█░▄▄▀█░▄▄█░▄▄█░██░█░▄▄
+                                                ██░█▀▀█░▄▄█░██░█░▄▄█▄▄▀█░▀▀░█▄▄▀
+                                                ██░▀▀▄█▄▄▄█▄██▄█▄▄▄█▄▄▄█▀▀▀▄█▄▄▄
+                                                ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <Provider
+    Name="Genesys"
+    DisplayName="Genesys Cloud"
+    Id="2e34b9d0-5c1f-4e23-9ac9-b4aefde213d2"
+    Documentation="https://developer.genesys.cloud/">
+
+    <Environment Issuer="{settings.Issuer}">
+      <Configuration AuthorizationEndpoint="{settings.Issuer.Scheme}://login.{settings.Issuer.Host}/oauth/authorize"
+                     TokenEndpoint="{settings.Issuer.Scheme}://login.{settings.Issuer.Host}/oauth/token"
+                     UserInfoEndpoint="{settings.Issuer.Scheme}://api.{settings.Issuer.Host}/api/v2/users/me">
+        <CodeChallengeMethod Value="plain" />
+        <CodeChallengeMethod Value="S256" />
+
+        <GrantType Value="client_credentials" />
+        <GrantType Value="authorization_code" />
+        <GrantType Value="refresh_token" />
+        <GrantType Value="implicit" />
```

Comment:
Let's remove this one: Genesys only seems to support the insecure OAuth 2.0-only implicit flow flavor and not the OIDC version (which is safer).

Reply:
this is what the server supports:

![image](https://github.com/user-attachments/assets/3347075f-b039-4098-85dc-47960a7581bb)